ts/de_DE.ts

```diff
@@ -317,7 +317,7 @@
     <message>
         <location filename="gui.py" line="1268"/>
         <source>In what units you want to scale?</source>
-        <translation>In welcher Einheit möchtest du skalieren?</translation>
+        <translation>In welcher Einheit möchten Sie skalieren?</translation>
```

Comment:
Well I guess I'm too young to respect that and I do work on international projects with the "you" in english too often. It feels very strange to use formal addressing for a change now and then and I'm sort of trying to push everyone towards the informal addressing, just so much easier to handle in every context except business relations with older people :)